Bottom: ddbb4a7e24675ea08e1c7668fabdd6c39268d374
Top:    0bd493e05ee07e44e9f7aedbc669d49d89244b84
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2012-12-13 01:56:57 +0000

borast: Win32: Comment #define in borast-compiler-private.h


---

diff --git a/src/borast/borast-compiler-private.h b/src/borast/borast-compiler-private.h
index f886f40..165b42e 100644
--- a/src/borast/borast-compiler-private.h
+++ b/src/borast/borast-compiler-private.h
@@ -160,7 +160,7 @@
 #endif
 
 #if (defined(__WIN32__) && !defined(__WINE__)) || defined(_MSC_VER)
-#define snprintf _snprintf
+//#define snprintf _snprintf
 #define popen _popen
 #define pclose _pclose
 #define hypot _hypot
